examples/src/examples/misc/gizmos.example.mjs

```diff
@@ -122,7 +122,8 @@ data.set('camera', {
 });
 const camera = new pc.Entity('camera');
 camera.addComponent('camera', {
-    clearColor: new pc.Color(0.1, 0.1, 0.1)
+    clearColor: new pc.Color(0.1, 0.1, 0.1),
+    farClip: 1000
```

Comment:
Why won't the example work with the default far clip?

Reply:
It does it you can just see the the axis lines clipping when zooming out